index.js

```diff
@@ -34,25 +34,42 @@ app.use(websocket());
 app.use(bodyParser({ enableTypes: ['json', 'text'], strict: false }));
 app.use(async (ctx) => {
   if (ctx.method === 'POST' && ctx.path === '/') {
-    const contents = isEmptyObject(ctx.request.body) ? '' : ctx.request.body;
+    const {contents} = ctx.request.body;    
+
+    //creates various id 
     const docId = uuid();
-    const doc = db.connect(undefined, docId).get(COLLECTION_NAME, docId);
+    const sessionEditingId = uuid().slice(0,6);
+    const sessionViewingId = uuid().slice(0,6);
+
+    const connection = db.connect(undefined, docId);
+    const doc = connection.get(COLLECTION_NAME, docId);
     await new Promise((resolve, reject) => {
-      doc.create(contents, (err) => {
+      doc.create({contents}, (err) => {
         if (err) {
           reject(err);
         } else {
           resolve();
         }
       });
     });
-    documents.add(docId);
-    ctx.body = docId;
+
+    const sessionDetails = new Map([
+      ["docId", docId], 
+      [sessionEditingId, false], 
+      [sessionViewingId, true]
+    ]);
+    documents.add(sessionDetails);
+
+    console.log(documents);
```

Comment:
Can we remove the `console.log` statements once they're no longer used for debugging?